doc/INSTALL.md

```diff
@@ -134,20 +187,6 @@ or in the build directory (will not honor `DESTDIR`!):
 xargs rm < install_manifest.txt
 ```
 
-### CPack
-
-First follow the steps in [COMPILE](COMPILE.md).
```

Comment:
Maybe we should keep this info somewhere?

Reply:
It is already present in the parent heading `## OS Independent` (not visible in the git diff). Is it really necessary to write this multiple times?

Reply:
Yes, you are right, I did not see it.